.github/workflows/bazel_build_bindings.yml

```diff
@@ -46,6 +46,9 @@ jobs:
           # Update pip before using it so it picks up the latest versions of our dependencies.
           sudo python3 -m pip install --upgrade pip
           sudo python3 -m pip install numpy tf-nightly
+          sudo apt-get install -y python-pip
+          python -m pip install --upgrade pip
+          python -m pip install numpy
```

Comment:
Can you add a comment to each file for why this is necessary, linking to the issue?

Reply:
Yeah!